src/visGeometry/index.ts

```diff
@@ -841,7 +849,10 @@ class VisGeometry {
         return this.threejsrenderer;
     }
 
-    public reparent(parent?: HTMLElement | null): void {
+    public setCanvasOnTheDom(
```

Comment:
I changed the name of this function because it's doing more than reparenting. If you have a better name please let me know

Reply:
Really like the name change. Fixing the viewport once on mount seems good for now, but I can definitely imagine a use case for exposing `setupControls` for reactive/ongoing configuration.